package.json

```diff
@@ -34,60 +34,55 @@
   "devDependencies": {
     "body-parser": "^1.17.1",
     "broccoli-asset-rev": "^2.4.5",
-    "broccoli-merge-trees": "^2.0.0",
```

Comment:
Hm, maybe old ESA dependency, pretty sure it's not needed though. Will check again.